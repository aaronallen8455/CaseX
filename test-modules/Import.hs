module Import
  ( Foo(..)
  ) where

data Foo = Foo | Bar | Baz
