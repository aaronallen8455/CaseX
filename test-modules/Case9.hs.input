import qualified Import2 as I
import           Import

main :: IO ()
main = pure ()

test :: Foo -> Bool
test x = case x of
  SPLIT -> True
