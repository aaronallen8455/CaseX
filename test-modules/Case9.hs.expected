import qualified Import2 as I
import           Import

main :: IO ()
main = pure ()

test :: Foo -> Bool
test x = case x of
  Foo -> True
  Bar -> True
  Baz -> True
